backstop/tests/tile.html

```diff
@@ -568,6 +559,21 @@ <h2>Default</h2>
               />
             </svg>
           </button>
+
+          <div class="iui-metadata">
```

Comment:
Why you need to switch places?

Reply:
Yeah, looks really weird, knowing that metadata should go before options menu...

Reply:
it's either this or I can use `:last-child`

But right now, the metadata width decreases on hover even when there's no options menu

Reply:
I changed to `last-child` but it won't work when user has custom children (which get added after metadata and more options)